examples/with-particles/pages/api/hello.js

```diff
@@ -0,0 +1,5 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```

Comment:
I think this file can be removed as it does not contribute to showcase the library.

Reply:
Removed

Reply:
> Thanks, I added some small comments.
> 
> Another question, what's the correct way of referring to the library? I see the directory was named `with-particles`, but the package is called `react-tsparticles`, the website https://particles.js.org/ says "TS Particles" (while the page title is tsParticles).
> 
> The reason I'm asking is that there is a https://vincentgarreau.com/particles.js/ so to not confuse developers, maybe we should call the directory `with-tsparticles`? What do you think?

`react-particles-js` was using `tsParticles` in its core since `3.0.0` but it was deprecated some weeks ago in favor of `react-tsparticles`. Using `react-particles-js` is not recommended, since it's not working anymore. I think the confusion could be avoided keeping `with-particles`.

These [CRA templates](https://github.com/matteobruni/tsparticles/tree/main/templates) are using `particles` as well (like `npx create-react-app my-app --template particles`)